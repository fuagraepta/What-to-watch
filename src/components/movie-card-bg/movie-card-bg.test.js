import React from 'react';
import renderer from 'react-test-renderer';

import MovieCardBg from './movie-card-bg';

describe(`The component is rendered correctly`, () => {
  it(`Movie-card-bg correctly renders with transferred mock data`, () => {
    const movieCardBg = renderer
    .create(<MovieCardBg
      bigPoster = {`img/bg-the-grand-budapest-hotel.jpg`}
      title = {`Johnny English`}
    />)
    .toJSON();
    expect(movieCardBg).toMatchSnapshot();
  });
});
