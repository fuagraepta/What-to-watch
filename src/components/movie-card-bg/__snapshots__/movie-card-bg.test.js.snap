// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The component is rendered correctly Movie-card-bg correctly renders with transferred mock data 1`] = `
<div
  className="movie-card__bg"
>
  <img
    alt="Johnny English"
    src="img/bg-the-grand-budapest-hotel.jpg"
  />
</div>
`;
